### ISSUE TITLE
Wizards that sell TFSP spells use base mod texture for book

### ISSUE THREAD

**Issue by TherminatorX** [Electroblob77/TF-Spell-Pack#10]
Version 1.0, title spells it out but here's a picture. This is not corrected upon purchase either.
![image](https://user-images.githubusercontent.com/7256249/81222458-ed32dd00-8fa9-11ea-9b7a-97f07c02a6bf.png)

**Comment by Electroblob77** [comment]
Wizards shouldn't really be selling twilight spells at all... this is probably a bug in wizardry itself rather than the TF spell pack. You should still be able to use the book though.

**Comment by TherminatorX** [comment]
I am able to use the spell books to attune wands, but if it's not intended for wizards to sell them perhaps that should be another spell trait, if they can be sold by wizards or not. Addon spells certainly benefit from the ability to be sold.
However, I've been having absolute rotten luck locating Twilit Tomes(tm) in the structures (only 3 after progressing through all of TF), so this does provide a nice way to get them otherwise.

**Comment by Electroblob77** [comment]
Agreed, what I need to do is add a hook for addon mods to register their spell books as part of wizard trades if they want to.

That really is unlucky though, I remember finding 3 in a single lich tower during testing. You can also get them as drops from the druid mages sometimes, which might help you find some more. I may reduce the rarity a bit though, they are still quite rare compared to regular spell books.

**Comment by TherminatorX** [comment]
I also haven't found a druid mage, though I've been on the lookout for them. I'll continue to explore the other areas beyond the 'main' Twilight map as I'm on a server and wouldn't want to hinder the exploration of others.
